resources/views/dashboard/individual.blade.php

```diff
@@ -16,6 +16,12 @@
                 </li>
             @endcan
         @endif
+        @if (!$user->oriented_at)
+            <li>
+                <a
+                    href="https://share.hsforms.com/161eyaBsQS-iv1z0TZLwdQwdfpez">{{ __('Sign up for an orientation session') }}</a>
```

Comment:
I think you forgot to use your orientation helper function in many of the spots.